README.md

```diff
@@ -3,7 +3,7 @@ Kashoo IAP Tester
 =================
 
 Created by Ben Kennedy on 2013-03-22.
-Copyright (c) 2013 Kashoo Inc. All rights reserved.
+Copyright (c) 2016 Kashoo Inc. All rights reserved.
```

Comment:
The copyright really dates back to 2013, if the newer date is warranted, then “2013–2016” would be more expressive, no?